docs/buildah-bud.md

```diff
@@ -40,9 +40,12 @@ Note: this information is not present in Docker image formats, so it is discarde
 
 **--authfile** *path*
 
-Path of the authentication file. Default is ${XDG\_RUNTIME\_DIR}/containers/auth.json, which is set using `buildah login`.
+Path of the authentication file. By defalut, the authication storage is the kernel keyring. If the system does not support kernel keyring, buildah will use the authentication file. Default is ${XDG\_RUNTIME\_DIR}/containers/auth.json, which is set using `buildah login --authfile nokeyring`.
 If the authorization state is not found there, $HOME/.docker/config.json is checked, which is set using `docker login`.
 
+Note: You can also override the default path of the authentication file by setting the REGISTRY\_AUTH\_FILE
```

Comment:
No "You".  "The default path of the authentication file can also be overridden by setting the REGISTRY\_AUTH\_FILE"